sci-misc/boinc/boinc-9999.ebuild

```diff
@@ -29,7 +29,7 @@ REQUIRED_USE="^^ ( curl_ssl_gnutls curl_ssl_openssl ) "
 
 # libcurl must not be using an ssl backend boinc does not support.
 # If the libcurl ssl backend changes, boinc should be recompiled.
-COMMON_DEPEND="
+DEPEND="
 	acct-group/boinc
```

Comment:
I am not sure if we have a policy on this, but it is unnecessary to explicitly state the dependency on the user's `acct-group` package, as `acct-user/boinc` already depends on `acct-group/boinc`. Maybe a good opportunity to fix in the live ebuild and the revbumped ebuild.

Reply:
Hasn't been merged yet and I'm at my dev PC...

Reply:
This was just briefly discussed in #gentoo-qa, there doesn't appear to be a policy nor consensus about this. So feel free to keep as is.